modules/sdk/ant-bnd/src/main/java/com/liferay/ant/bnd/jsp/JspAnalyzerPlugin.java

```diff
@@ -562,8 +564,24 @@ protected boolean matchesURI(
 		return false;
 	}
 
-	private boolean _isUseJavaxImports(Set<String> taglibURIs) {
+	private boolean _isUseJavaxImports(
+			String javaeePackage, Set<String> taglibURIs)
+		throws Exception {
+
 		if (taglibURIs.isEmpty()) {
+			if (javaeePackage != null) {
+				if (javaeePackage.equals("javax")) {
+					return true;
+				}
+				else if (javaeePackage.equals("jakarta")) {
+					return false;
+				}
+
+				throw new RuntimeException(
+					"Value for -javaee-package is invalid; valid values are " +
+						"jakarta or javax");
```

Comment:
detail here exactly what the situation is to the developer:
- Could not infer
- Was not given a fallback value